flake.nix

```diff
@@ -183,8 +190,20 @@
           packages = [sonataSystemPkgs.sonata-simulator] ++ default.nativeBuildInputs;
           SONATA_SIM_BOOT_STUB = "${sonataSystemPkgs.sonata-sim-boot-stub.out}/share/sim_boot_stub";
         };
+        env-with-everything = pkgs.mkShell {
+          name = "sonata-sw-with-everything";
+          packages = env-with-sim.nativeBuildInputs ++ [lrPkgs.lowrisc-toolchain-gcc-rv32imcb];
+        };
```

Comment:
This works for me, but would it be possible to just have a `legacy` environment or equivalent that does not include the simulator build? It its quite a lengthy process to build the sim, and as far as I'm aware this isn't needed to build legacy baremetal software.

Reply:
I just wanted to avoid having too many environments, so I thought I'd make a catchall everything environment. But I suppose people can always overlay environments, open to changing